integration-test-groups/aws2/aws2-cw/src/main/java/org/apache/camel/quarkus/component/aws2/cw/it/Aws2CwResource.java

```diff
@@ -61,4 +68,48 @@ public Response post(
                 .created(new URI("https://camel.apache.org/"))
                 .build();
     }
+
+    @Path("/send-metric-map/{namespace}")
+    @POST
+    @Consumes("application/x-www-form-urlencoded")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response postMap(
+            @PathParam("namespace") String namespace,
+            @HeaderParam("customClientName") String customClientName,
+            MultivaluedMap<String, String> formParams) throws Exception {
+
+        String uri = "aws2-cw://" + namespace;
+        uri = customClientName != null && !customClientName.isEmpty()
+                ? uri + "?autowiredEnabled=false&amazonCwClient=#" + customClientName : uri;
+
+        //use Instant as timestamp header
```

Comment:
Does this comment apply to the whole set of instructions below ?

Reply:
removed